File: discover_ruby/github_most_popular.rb
require 'httpclient'

extheaders = {
  'User-Agent' => 'Holberton_School',
  'Authorization' => 'token aa691ecddbb46f017092bdb87a3c55300bd33e9d'
 }  


clnt = HTTPClient.new

puts clnt.get_content('https://api.github.com/search/repositories?q=language:ruby&sort=stars&order=desc', query = nil, extheaders = {})




 #uri = URI('https://api.github.com/search/repositories?q=language:ruby&sort=stars&order=desc')
     #res = Net::HTTP.get(uri)
#uts res     

File: discover_more_python/github_most_popular.py
import urllib2

request_headers = {
  'User-Agent': 'Holberton_School',
  'Authorization': 'token aa691ecddbb46f017092bdb87a3c55300bd33e9d'
}

r = urllib2.urlopen("https://api.github.com/search/repositories?q=language:python&sort=stars&order=desc").read()

print r

